xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -670,6 +672,18 @@ bool CDecoder::Open(AVCodecContext* avctx, AVCodecContext* mainctx, const enum A
       if (!m_vaapiConfig.context->SupportsProfile(profile))
         return false;
       break;
+    case AV_CODEC_ID_AV1:
+    {
+      if (avctx->profile == FF_PROFILE_AV1_MAIN)
+        profile = VAProfileAV1Profile0;
```

Comment:
Might be worth checking what version of libva this was added in as we might have to bump the minimum version of add guards

Reply:
av1 decoding was added with libva-2.7.0:
https://github.com/intel/libva/blob/master/NEWS#L77
https://github.com/intel/libva/commit/7eb84e41658944ab0713c5aa6b8eddbcb50a114d

Reply:
I used the same VA_CHECK_VERSION that is used inside ffmpeg, which is 1.8.0.

Reply:
Interestingly those defines seem to work for our freebsd builder, which complained about missing AV1 defines and works now.
I'll have a closer look once again.

Reply:
sorry, I took another look. API version is disconnected from lib version. lib 2.7.0 has API version 1.7.0. I guess ffmpeg depends on some later fixes to require API version 1.8.0. I think this PR is ready.

Reply:
Yeah just confirmed the same.